tools/functional-tester/etcd-tester/tester.go

```diff
@@ -83,6 +83,13 @@ func (tt *tester) runLoop() {
 			plog.Printf("[round#%d case#%d] recovered failure", i, j)
 
 			if !tt.consistencyCheck {
+				plog.Printf("[round#%d case#%d] restarting the stressers...", i, j)
```

Comment:
i mean why do we need to stop them and restart immediately? this does not make sense to me.